### DISCUSSION TITLE
Is it possible to ignore inline gql tags?

### DISCUSSION THREAD

**Discussion by richardgirges** [dotansimha/graphql-code-generator#7645] [General]
Hi gql codegen team. I apologize if this question has been answered, I looked around past tickets and through the docs but couldn't find an answer.

Is it possible to force codegen to skip certain inline queries? 

Example:

```typescript
// graphql-codegen-ignore-next-line
const query = gql`
  query Foo { foo }
`;
```

If that's not possible, are there any other alternatives for ignoring queries on a per-case basis? 

Thanks!

**Comment by cesarvarela** [comment]
I need this too, any news?
